opentelemetry-appender-log/src/lib.rs

```diff
@@ -0,0 +1,61 @@
+use log::{Level, Metadata, Record};
+use opentelemetry_api::logs::{AnyValue, LogRecordBuilder, Logger, LoggerProvider, Severity};
+
+pub struct OpenTelemetryLogBridge<P, L>
+where
+    P: LoggerProvider<Logger = L> + Send + Sync,
+    L: Logger + Send + Sync,
+{
+    logger: L,
+    min_level: Level,
```

Comment:
I think in order for this to useful, the level can't be fixed at construction. Instead, the enabled check should query something like the sampler to see if the event is enabled. Otherwise systems that can change the collected level dynamically (e.g. ETW, user_events\eventheader) won't be able to function well unless the level here is always fixed at the "collect everything" value, which really defeats the whole point of having a level to begin with.